project_euler/euler_prob_2/euler2.rb

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
You committed a lot of things here that don't have anything to do with the problem - euler 4.  It's worth looking at whats about to be committed before you commit - do these go in other branches?